index.js

```diff
@@ -177,6 +184,45 @@ function seededShuffle( array, salt ) {
 	return shuffledArray;
 }
 
+function integerSequence( from, to ) {
+	var list = [];
+	var step = Math.sign( to - from );
+
+	for (var i = from; i <= to; i += step) {
+		list.push(i);
+	}
+
+	return list;
+}
+
+function seededUnShuffle( array, salt ) {
+	var         sequence = integerSequence(0, array.length - 1);
+	var shuffledSequence = seededShuffle( sequence, salt );
+	var shuffledIndicies = [];
+
+	var i;
+
+	for (i = 0; i < shuffledSequence.length; i++) {
+		shuffledIndicies[shuffledSequence[i]] = i;
+	}
+
+	var  unShuffledArray = [];
+
+	for (i = 0; i < shuffledSequence.length; i++) {
+		unShuffledArray[i] = array[shuffledIndicies[i]];
+	}
+
+	return unShuffledArray;
+}
+
+function calcIndiciesChecksum( indicies ){
+	return indicies.reduce(function(prev,current){ return prev+current; }, 0);
+}
+
+function calcChecksumAsIndex( checksum, modulus ){
+	return mod(checksum, modulus);
```

Comment:
Seems like a pointless function as it is just passing the same parameters in the same order.

This could be rewritten as `var calcChecksumAsIndex = mod;`, which would highlight the point that it is just a renaming of a function.